Frontend/ui-driver/index.js

```
@@ -446,12 +490,16 @@ if (sessionInfo.package_name.includes(".debug") || sessionInfo.package_name.incl
   window.Android.runInUI("android.webkit.WebView->setWebContentsDebuggingEnabled:b_false;", "null");
 }
 
-JOS.emitEvent("java","onEvent",JSON.stringify({ action: "DUI_READY", event: "initiate",service : JOS.self }))();
+JOS.emitEvent("java", "onEvent", JSON.stringify({
+  action: "DUI_READY",
+  event: "initiate",
+  service: JOS.self
+}))();
 
 if (!window.JOS.tracker) {
   eval(window.JBridge.loadFileInDUI("v1-tracker.jsa"));
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)